test/components/DateInput_spec.jsx

```diff
@@ -97,6 +97,35 @@ describe('DateInput', () => {
         });
       });
     });
+
+    describe('aria-labelledby', () => {
+      let wrapper;
+      const inputId = 'date';
+      const ariaLabelledBy = 'id1 id2';
+
+      describe('props.ariaLabelledBy is truthy', () => {
+        beforeEach(() => {
+          wrapper = shallow((
+            <DateInput id={inputId} ariaLabelledBy={ariaLabelledBy} />
+          )).dive();
+        });
+
+        it('has aria-labelledby attribute that starts with ariaLabelledBy value', () => {
+          expect(wrapper.find(`input[aria-labelledby="${ariaLabelledBy}"]`)).to.have.lengthOf(1);
+        });
+      });
+
+      describe('props.ariaLabelledBy is falsy', () => {
+        beforeEach(() => {
+          wrapper = shallow(<DateInput id={inputId} />).dive();
+        });
+
+        it('does not have aria-labelledby attribute value', () => {
+          expect(wrapper.find(`input[aria-labelledby="${ariaLabelledBy}"]`)).to.have.lengthOf(0);
```

Comment:
You can simplify these two lines by doing `wrapper.find('input[aria-labelledby]')` instead

Reply:
When I try that, the `.find` does return one item bc the property is there in the React component, so I don't think this will work? Maybe I'm doing it wrong?

Reply:
Oh, I think it might be because you're passing `null` and not `undefined`. If I remember correctly, passing `null` will result in `null` getting passed as the value to that prop and passing `undefined` will not set the prop. This is accomplishing what you want so no need to change it!